web/src/components/Board.res

```diff
@@ -177,7 +177,11 @@ module Board = {
       {
         Column.name: "To Review",
         order: mkOrder("created_at", None),
-        query: "state:open and updated_at>now-3week " ++ " and not " ++ defaultNegativeApprovals,
+        query: "state:open and updated_at>now-3week " ++
+        " and not " ++
+        defaultNegativeApprovals ++
+        " and not " ++
```

Comment:
I think you meant:
```suggestion
        " and " ++
```

Reply:
The idea is to not see those already in the "to approve" section.

Reply:
But well if you don't like the change I can revert and keep this only for my dashboard.